labkey-client-api/src/org/labkey/remoteapi/di/ResetTransformStateResponse.java

```diff
@@ -34,7 +30,7 @@ public class ResetTransformStateResponse extends BaseTransformResponse
      * @param json          The parsed JSONObject (or null if JSON was not returned).
      * @param sourceCommand A copy of the command that created this response
      */
-    public ResetTransformStateResponse(String text, int statusCode, String contentType, JSONObject json, Command sourceCommand)
+    public ResetTransformStateResponse(String text, int statusCode, String contentType, JSONObject json, Command<?> sourceCommand)
```

Comment:
I haven't tested this locally, but I don't see a reason why the last parameter in this constructor shouldn't be `ResetTransformStateCommand`. Same with a bunch of other `*Response` constructors.

Reply:
I don't think we want arbitrary `Command` objects being passed in

Reply:
I updated all of the response classes to take `Command<? extends ThisResponseClass> sourceCommand` for maximum extensibility; it was required for some commands/responses, so I made the rest match.